scaling_performance/optimizing_storage.adoc

```diff
@@ -304,6 +323,8 @@ Storage Driver: devicemapper
  Metadata Space Total: 16.78 MB
  Metadata Space Available: 16.74 MB
 ----
+<1> The `docker info` output when using `devicemapper`.
+<2> Corresponds with the `VG` you specified in *_/etc/sysconfig/docker-storage-setup_*
```

Comment:
s/with/to
s/\*_\/etc/sysconfig/docker-storage-setup\_\*/the \*\_/etc/sysconfig/docker-storage-setup\_\* file.